src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs

```diff
@@ -41,7 +41,8 @@ static CommonGeneratorMethods()
         Name = "gzip",
         Method = serviceProvider => (Func<string, string>)(encoded =>
         {
-            var compressedStream = Base64.DecodedToStream(encoded);
+            var compressedStream = Base64.DecodeToStream(encoded);
+            compressedStream.Seek(0, SeekOrigin.Begin);
```

Comment:
Why? the DecodeToStream method is now doing it

Reply:
Looks like a merge issue as this is what the other PR removed

Reply:
the Other PR added this "compressedStream.Seek(0, SeekOrigin.Begin)"

Reply:
wait! how did this show up here.

Reply:
It added it, not there

Reply:
@sebastienros I removed it. You good with this?